src/style.css

```diff
@@ -1,3 +1,79 @@
+/* Moyo */
+
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+ul {
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: white;
```

Comment:
Better is to use HEX, cause in different browsers color names contain different colors.